about/policy/data-sharing-agreement/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: State Geographic Information Database (SGID) Data Sharing Agreement
+layout: page_nocrumb
+author:
+  display_name: UGRC Office
+  email: ugrc@utah.gov
+tags:
+  - license
+  - policy
+categories: []
+date: 2021-07-16 17:00:00 -0700
+---
+
+## Applicability
+{: .text-left}
+
+The following Data Sharing Agreement applies to all individuals and organizations sharing data with the SGID in any way.
+
+In order to reduce the friction of adding data to the SGID, by submitting a [Porter issue](https://github.com/agrc/porter/issues) to introduce your data to the SGID, you consent to this Data Sharing Agreement. In cases where UGRC submits the Porter issue on your behalf, UGRC will contact you via email to get your consent to this agreement.
+
+## Definitions
+{: .text-left}
+
+**"Aggregate data"** refers to any data that cover a specific area of the state and are provided by another organization (such as a city or county) for UGRC to combine with other similar data into statewide datasets.
```

Comment:
are we planning on reaching out to our existing county stewards about this, or just putting this out there as a place to point folks to if questions arise?

Reply:
i believe jake mentioned this was intended for new data

Reply:
i like that plan

Reply:
Yes to both. A lot of our existing coordination is probably governed by statute or specific agreements, and technically this can be interpreted to just apply to new datasets (though the line at the beginning does reference everything). I view it as similar to our CC license- we're just explicitly stating our intents to maintain good data.

Reply:
nice.  i like it.